zero_bin/common/src/block_interval.rs

```diff
@@ -7,8 +10,13 @@ use futures::Stream;
 use tracing::info;
 
 use crate::parsing;
+use crate::provider::CachedProvider;
+
+// Time in milliseconds to poll the node for new blocks.
+// We set it to 12 seconds, default ETH block time.
+const DEFAULT_BLOCK_TIME: u64 = 12000;
```

Comment:
this is fine, _for now_, but this would need to be parameterizable ideally.
zkEVM has a 3sec blocktime, IMX and Polygon mainnet have a 2sec block time, etc...

Reply:
It is already parametrized with CLI argument `block_time`. Will remove this one as it is not used.